docs/tutorials/typescript/digital-ocean-deployment/index.mdx

```diff
@@ -0,0 +1,283 @@
+  - Temporal Cloud certificates
+---
+
+When you are ready to deploy your TypeScript based Temporal Application to production, you need a server to host it.
+Digital Ocean is provides flexible Cloud servers called Droplets that can be used to host your application.
+
+In this tutorial we will walk through the steps to deploy your Temporal Application to a Digital Ocean Droplet while using Temporal Cloud as your Temporal Application's supervisor.
+
+## Create a Droplet {#create-droplet}
+
+Assuming you already have a Digital Ocean Account, create a new Droplet and choose the Ubuntu 20.04 image or Ubuntu 22.04 image.
+
+**Set up SSH**
+
+Next, set up SSH so you can set up your server from your local terminal.
+You can do that by following the [Add SSH Key to Droplet tutorial](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/).
+
+If you already have a private key on your local machine and you need to force SSH to use the new key, you can force it with the `-i` flag, for example:
+
+```
+ssh -i ~/.ssh/id_rsa_digitalocean
+```
+
+**Install Node.js**
+
+Once you have SSH'd into your Droplet, you can install Node.js.
+
+Follow these tutorials, for Ubuntu 20.04 or 22.04 respectively:
+
+- [How To Install Node.js on Ubuntu 20.04](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04)
+- [How To Install Node.js on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-22-04)
+
+**Install TypeScript and TS Node**
+
+```
+sudo npm install -g typescript ts-node
+```
+
+## Use a custom domain {#custom-domain}
+
+You need a domain, because you need to have a valid SSL certificate for the Temporal Application to communicate with Temporal Cloud.
+
+Register your domain and ensure the domain's A record points to the droplet's IP address.
+
+## Set up Nginx as a reverse proxy {#set-up-nginx}
+
+SSH into your Droplet and set up Nginx as a reverse proxy.
+
+Follow these tutorials, for Ubuntu 20.04 or 22.04 respectively, including setting up the server blocks:
+
+- [Set up Nginx for Ubuntu 20.04](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-20-04)
+- [Set up Nginx for Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-as-a-reverse-proxy-on-ubuntu-22-04)
+
+## Create a domain certificate {#create-domain-cert}
+
+Use Let's encrypt certbot to create a certificate for your domain.
+This will effectively enable HTTPS for your domain and your Digital Ocean Droplet.
+
+Follow the tutorial for your Ubuntu version:
+
+- [How To Secure Nginx with Let's Encrypt on Ubuntu 20.04](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-20-04)
+- [How To Secure Nginx with Let's Encrypt on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-22-04)
+
+## Create a Temporal Cloud certificate {#create-temporal-cloud-cert}
```

Comment:
```suggestion
## Step 3 -- Create a Temporal Cloud certificate {#create-temporal-cloud-cert}
```

This section is pretty good, well-motivated, concise, and links out to other content of ours. All good things.